src/ppx_deriving.ml

```diff
@@ -2,6 +2,7 @@ open Longident
 open Location
 open Asttypes
 open Parsetree
+open Ast_mapper
```

Comment:
They are both used multiple places in `ppx_deriving`, esp. I would prefer to keep Ast_mapper due to the record field uses.  OTOH, I could remove all open statements from `ppx_deriving_main`.